src/utils/api.js

```diff
@@ -56,17 +59,34 @@ export const handleLogin = async (user) => {
 export const handleLogout = async () => {
   try {
     const url = `${rootUrl}/api/v1/auth/logout`
-    const response = await fetch(url)
+    const response = await fetch(url, { credentials: 'include' })
```

Comment:
I mention in the video, that you might see your backend start to return errors when you add the `{ credentials: 'include' }` step. Specifically in the browser console, you might see error messages that are something like : `CORS: Cannot use wildcard in Access-Control-Allow-Origin when credentials flag is true`.

This is because in the backend, we initially just used the default values for the `cors` middleware: https://github.com/DYA13/e-commerce/blob/main/app.js#L40

You'll want to update that to something like: `app.use(cors({ origin: ["http://localhost:_whatever_port_you_use_/", "https://whatever_your_production_deployed_domain_will_be.com/"], credentials: true }));`

This modifies the [cors](https://www.npmjs.com/package/cors) middleware handled by the backend
* [origin](https://www.npmjs.com/package/cors#configuration-options): this sets the Access-Control-Allow-Origin HTTP header returned by the server. This tells the browser that it’s okay to make cross-domain requests to this list of hosts. Before, when we didn’t define the list, the server was returning a wildcard (*) which tells the browser, “you can send cross-domain requests to any host you want”.
	* BUT, once we added withCredentials: true  in our React code so that Axios would pass along Cookies to the server, then this stops working because browsers nowadays have a safeguard which makes them not allow wildcard cross-origin requests if you’re passing around cookies.
	* This is good security practice to prevent hackers from being able to read or set cookies from any random URL (more info about this header in [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin))
* [credentials](https://www.npmjs.com/package/cors#configuration-options): this sets the Access-Control-Allow-Credentials HTTP header returned by the server. This tells the browser that it’s okay to pass along cookies to Javascript code (more info about this header in [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Credentials))

The URLs you put in there should be the URL you usually run your React code from locally, and the URL the react code runs on when deployed